### PR TITLE
Update php8.2 compatible changes

### PR DESCRIPTION
**Pull request by imranbspr**
Changes are on beta https://github.com/spreaker/www-site/pull/7517.

### CODE REVIEW

File: src/Facebook/FacebookApp.php [hunk 1]
```diff
@@ -89,21 +89,21 @@ public function getAccessToken()
...
      * Serializes the FacebookApp entity as a string.
      *
-     * @return string
+     * @return array
      */
-    public function serialize()
+    public function __serialize(): array
     {
-        return implode('|', [$this->id, $this->secret]);
+        return [$this->id, $this->secret];
```
Comment (micheleorselli) [inline src/Facebook/FacebookApp.php:96]: not sure this is important in that specific case but if we have serialized data saved somewhere this is gonna break

Comment (imranbspr) [reply src/Facebook/FacebookApp.php:96]: If we keep `implode`, the return type will mismatch.

Comment (micheleorselli) [reply src/Facebook/FacebookApp.php:96]: I guess we have to test it with the new implementation and see if something breaks

**Review by micheleorselli — APPROVED** [review]

**Review by dymissy — APPROVED** [review]
Unless there's any magic inside the code, it looks like the serialize/unserialize methods are not being used.

Do we want to go with a YOLO approach? :trollface:

**Pull request merged**